src/uucore/src/lib/mods/posix.rs

```diff
@@ -0,0 +1,52 @@
+// This file is part of the uutils coreutils package.
```

Comment:
if uniq is the only user of this function, isn't it a bit overkill to use have it in uucore?